packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+
+import {IButtonProps} from '../button/Button';
+import {InputConnected} from '../input/InputConnected';
+import {multilineBoxWithRemoveButton} from '../multilineBox/hoc/MultilineBoxWithRemoveButton';
+import {IMultilineParentProps, IMultilineSingleBoxProps, MultilineBox} from '../multilineBox/MultilineBox';
+
+const MultilineBoxWithRemoveButton = _.compose(
+    multilineBoxWithRemoveButton({
+        containerNode: (
+            child: React.ReactNode,
+            getRemoveButton: (props?: Partial<IButtonProps>) => React.ReactNode,
+            data: Array<IMultilineSingleBoxProps<string>>,
+            index: number
+        ) => (
+            <div key={`${data[index].id}`} className="flex">
+                <div className="flex-auto">{child}</div>
+                {getRemoveButton({
+                    classes: ['mod-no-border', 'bg-transparent', 'align-end'],
+                })}
+            </div>
+        ),
+    })
+)(MultilineBox);
+
+export interface MultiValuesInputProps {
+    id: string;
+    data: string[];
+}
+
+export const MultiValuesInput: React.FunctionComponent<MultiValuesInputProps> = ({id, data}) => {
+    return (
+        <MultilineBoxWithRemoveButton
+            id={id}
+            data={data}
+            renderBody={(allData: Array<IMultilineSingleBoxProps<string>>, parentProps: IMultilineParentProps) =>
+                allData.map((cData: IMultilineSingleBoxProps<string>) => (
+                    <InputConnected
+                        key={cData.id}
+                        id={cData.id}
+                        defaultValue={cData.props}
+                        onKeyUp={(e: React.KeyboardEvent<HTMLInputElement>) => {
+                            const value = e.currentTarget.value;
+                            if (value !== '' && cData.isLast) {
+                                parentProps.addNewBox();
+                            }
+                        }}
+                        onBlur={(value: string) => {
+                            if (value === '' && !cData.isLast) {
+                                parentProps.removeBox(cData.id);
+                            }
+                        }}
```

Comment:
Do we need `validate` and `validateOnChange`, also `labelTitle` and `labelProps` for this `InputConnected`?

Reply:
I will add an `inputProps` to add everything that is missing